tools/build.md

```diff
@@ -29,25 +29,39 @@ within the manylinux2014 container are encouraged.
 
 We use the TFX manylinux docker that includes a working Bazel installation. Run
 the following command to start the docker (might require superuser permissions):
-```
+
+```sh
 ./tools/start_compile_docker.sh
 ```
+
 Within the docker, run the following command
+
+```sh
+PYTHON_VERSION=<version> ./tools/build_manylinux_all.sh
```

Comment:
Just build_manylinux.sh if you're specifying the python version.

Reply:
yep, typo, was supposed to be that one, good catch